.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "external/metal"]
```

Comment:
@willkill07 and @trws if we change over to BLT, which @davidbeckingsale and I are thinking we should (I'm working this into the perf suite now), things like googletest come along for free.